config/settings/base.py

```diff
@@ -370,5 +370,10 @@
 ADSERVER_HTTPS = False  # Should be True in most production setups
 ADSERVER_STICKY_DECISION_DURATION = 0
 
+# For customer support emails
+ADSERVER_SUPPORT_FROM_EMAIL = env("ADSERVER_SUPPORT_FROM_EMAIL", default=SERVER_EMAIL)
```

Comment:
No doc for this one?

Reply:
I'll remove it

Reply:
It actually wasn't even used. It always used [DEFAULT_FROM_EMAIL](https://docs.djangoproject.com/en/2.2/ref/settings/#std:setting-DEFAULT_FROM_EMAIL) which seems correct.